wd_helper.au3

```diff
@@ -976,6 +976,7 @@ EndFunc   ;==>_WD_ElementOptionSelect
 ;                  $sSelectElement - Element ID of Select element from _WD_FindElement
 ;                  $sCommand       - Action to be performed. Can be one of the following:
 ;                  |OPTIONS - Retrieve array containing value / label attributes from the Select element's options
+;                  |OPTIONS2 - Retrieve array containing value / label attributes from the Select element's options - using faster way - RegExp
```

Comment:
I suggest that we stick to a single 'Options' command. Either we replace the existing code with your version or perhaps there's a way to improve the existing version so that it doesn't loop through each option with WD commands.

Reply:
I agree with @TheDcoder  opinions
https://www.autoitscript.com/forum/topic/191990-webdriver-udf-w3c-compliant-version-01192022/?do=findComment&comment=1494624
`we also need to keep the current version as it is more robust, the RegEx might fail in edge cases, there's a saying about using RegEx with HTML`

Reply:
Yes, that makes perfect sense. Thanks for reminding me about that prior discussion

Reply:
I'm still unsure about the "options2" label. Can you come up with something that better describes this new command?